test/functional/expressions.test.ts

```diff
@@ -375,11 +375,11 @@ suite("Expressions functional tests", () => {
 
     suite("Miscellaneous and real scenarios", () => {
         testExpression("", "[concat(parameters('_artifactsLocation'), '/', '/scripts/azuremysql.sh', parameters('_artifactsLocationSasToken'))], )]", [
-            'Error: Nothing should exist after the closing \']\' except for whitespace. (ARM (Expressions)) [10,132-10,133]',
-            'Error: Nothing should exist after the closing \']\' except for whitespace. (ARM (Expressions)) [10,134-10,135]',
-            'Error: Nothing should exist after the closing \']\' except for whitespace. (ARM (Expressions)) [10,135-10,136]',
-            'Error: Undefined parameter reference: \'_artifactsLocation\' (ARM (Expressions)) [10,36-10,56]',
-            'Error: Undefined parameter reference: \'_artifactsLocationSasToken\' (ARM (Expressions)) [10,101-10,129]'
+            'Error: Nothing should exist after the closing \']\' except for whitespace. (arm-template (expr)) [10,132-10,133]',
+            'Error: Nothing should exist after the closing \']\' except for whitespace. (arm-template (expr)) [10,134-10,135]',
+            'Error: Nothing should exist after the closing \']\' except for whitespace. (arm-template (expr)) [10,135-10,136]',
+            'Error: Undefined parameter reference: \'_artifactsLocation\' (arm-template (expr)) [10,36-10,56]',
+            'Error: Undefined parameter reference: \'_artifactsLocationSasToken\' (arm-template (expr)) [10,101-10,129]'
```

Comment:
I'm guessing "expr" is PM call, I would've left "expression" since others are spelled out fully

Reply:
My call. Can be changed at any point.  It's just that "expressions" is longer.  Later we might add codes (e.g. "expr1" so we can track errors by type).